hugolib/page.go

```diff
@@ -168,30 +168,55 @@ func (p *Page) Param(key interface{}) (interface{}, error) {
 	return p.Site.Params[keyStr], nil
 }
 
+// Author returns the first listed author for a page
 func (p *Page) Author() Author {
-	authors := p.Authors()
+	authors := p.getAuthors()
+	if len(authors) == 0 {
+		return Author{}
+	}
+	return authors[0]
+}
+
+// Authors returns all listed authors for a page
+func (p *Page) Authors() AuthorList {
```

Comment:
The AuthorList here is a map and not a list and it should be a sorted slice with a well defined sort key (weight?)